src/ReanimatedModule.ts

```diff
@@ -0,0 +1,34 @@
+import { NativeEventEmitter, NativeModules } from 'react-native';
+import { nativeShouldBeMock } from './reanimated2/PlatformChecker';
+import { StyleProps } from './reanimated2';
+
+export const NODE_MAPPING = new Map();
```

Comment:
1. We don't want to export this outside the file. This is an internal module logic.
2. Can we rename `NODE_MAPPING` to something more descriptive?